web/static/js/reducers/user.js

```diff
@@ -2,9 +2,10 @@ const user = (state = [], action) => {
   switch (action.type) {
     case "SET_USERS":
       return action.users
-    case "UPDATE_USER":
+    case "UPDATE_USER": {
```

Comment:
Yeah, when you're setting a variable inside of a switch case the linter complains if you don't have it within its own block because each level of the switch will have access to the variables of other cases.